sdk/translation/ai-translation-text/src/customClient.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getClient, ClientOptions } from "@azure-rest/core-client";
+import * as coreRestPipeline from "@azure/core-rest-pipeline";
+import {
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest
+} from "@azure/core-rest-pipeline";
+import { TextTranslationClient } from "./generated/clientDefinitions";
+import {
+  TranslatorCredential,
+  TranslatorAuthenticationPolicy,
+  TranslatorAzureKeyAuthenticationPolicy
+} from "./authentication";
+import {
+  AzureKeyCredential,
+  TokenCredential
+} from "@azure/core-auth";
+
+const DEFAULT_SCOPE = "https://cognitiveservices.azure.com/.default";
+const DEFAULT_ENPOINT = "https://api.cognitive.microsofttranslator.com";
+const PLATFORM_HOST = "cognitiveservices";
+const PLATFORM_PATH = "/translator/text/v3.0";
+
+/**
+ * Initialize a new instance of `TextTranslationClient`
+ * @param endpoint type: string, Supported Text Translation endpoints (protocol and hostname, for example:
+ *     https://api.cognitive.microsofttranslator.com).
+ * @param options type: ClientOptions, the parameter for all optional parameters
+ */
+export default function createClient(
+  endpoint: undefined | string,
```

Comment:
If there is no such universal endpoint, could we remove `undefined` from the union?

Reply:
Replied in previous comment, it is valid prod endpoint.